tests/queries/0_stateless/03033_from_unixtimestamp_joda_by_int64.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+# NOTE: this sh wrapper is required because of shell_config
```

Comment:
What is exact purpose of making a `.sh` test file, if we can just write `select fromUnixTimestampInJodaSyntax(10262736196, 'YYYY-MM-dd HH:mm:ss', 'Asia/Shanghai')` in a `.sql` file?